doc/cookbook_fixers.rst

```diff
@@ -0,0 +1,561 @@
+==============================================
```

Comment:
`CONTRIBUTING.md` still links to the old version

Reply:
Fixed, thanks.